### PR TITLE
vamshi adep

### PR DESCRIPTION
**Pull request by rachhanari**

### CODE REVIEW

**Comment by rachhanari** [comment]
@bharat-dussa please review my code

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
 </head>
 <body>
     Raccha Narendra
+    vamshi adep
```
Comment (bharat-dussa) [inline index.html]: Change name to Nandini Ankam

File: index.html
Comment (bharat-dussa) [inline index.html:1]: change to Nandini

**Review by bharat-dussa — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (rachhanari) [reply index.html]: done

**Pull request merged**